### PR TITLE
build(installer): support for alpine linux

### PR DESCRIPTION
**Pull request by doshidhaval**
Fixes #[1374](https://github.com/stoplightio/spectral/issues/1374).

**Checklist**

- [ ] Tests added / updated - N/A
- [ ] Docs added / updated - N/A

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

**Additional context**
Implemented the fix based on @DenialAdams suggestion
Tested on Alpine Linux container, need to install the following packages for spectral to work on alpine
```
apk add libstdc++
apk add libgcc
```

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -14,6 +14,14 @@ if [ "$UNAME" = "Darwin" ] ; then
   FILENAME="spectral-macos"
 elif [ "$UNAME" = "Linux" ] ; then
   FILENAME="spectral-linux"
+  if [ -f /etc/os-release ]; then
+    # extract the value for KEY named "NAME"
+    DISTRO=$(sed -n -e 's/^NAME="\?\([^"]*\)"\?$/\1/p' /etc/os-release)
+    if [ "$DISTRO" = "Alpine Linux" ]; then
+      echo "Installing on Alpine Linux."
+      FILENAME="spectral"
```
Comment (P0lip) [inline scripts/install.sh:22]:
```suggestion
      FILENAME="spectral-alpine"
```

**Review by P0lip — COMMENTED** [review]
The filename is incorrect.
Otherwise looks good.

File: scripts/install.sh (hunk 1 above)
Comment (doshidhaval) [reply scripts/install.sh]: fixed

**Comment by P0lip** [comment]
Thanks for the PR!

**Pull request merged**